packages/react/src/composite/ScrollPane/CHANGELOG.md

```
@@ -2,6 +2,12 @@
 
 The ScrollPane component handles responsive scrolls in lists that are overflowing inside a container.
 
+## 2025-09-10 `1.0.1`
+
+#### 🎉 Others
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)